tools/dependencies/known-dependencies-jdk11.txt

```diff
@@ -507,7 +507,10 @@ metrics-core-3.2.1.jar
 metrics-core-4.0.0.jar
 minlog-1.3.0.jar
 mongo-java-driver-3.4.2.jar
+mongodb-driver-3.12.11.jar
+mongodb-driver-core-3.12.11.jar
 mongo-spark-connector_2.11-2.2.0.jar
```

Comment:
The newly added jar is not reflected in the license

Reply:
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/release-docs/LICENSE